### DISCUSSION TITLE
Database Reader

### DISCUSSION THREAD

**Discussion by PrabhakaranTankaey** [nextgenhealthcare/connect#4998] [Q&A]
Hi,
       I need to read data using database reader with 1 hour interval.i want read only new data,how to skip the already readed data.
       And what is the maximum limit of reading data using aggregate results.
       Thanks in advance .

**Comment by pacmano1** [comment]
You have to write code that decides if you read the data already.  One way is to update a column in the row itself.   Another is to have a separate table where you write a row that records the fact you processed a record.   With either method your database poller SQL can return the unprocessed records.  I assume based on your question that you cannot empty the table as you process it.

**Reply by pacmano1** [reply]
There are other interesting ways of course.  

A trigger on the table that inserts a row to a new table that you process and delete from.  

Publish / Subscribe to table level events is possible also.
